src/components/SearchInput/SearchInput.jsx

```diff
@@ -1,7 +1,62 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+
+import styles from './SearchInput.module.scss';
+import SearchIcon from '../../assets/images/search-icon.png';
+import SearchLocationIcon from '../../assets/images/search-location-icon.png';
+import { NUMBER_OF_RESULTS } from '../../constants/forecaApi';
+import { dataService } from '../../services/dataService';
 
 function SearchInput() {
-  return <div>Search input</div>;
+  const [searchQuery, setSearchQuery] = useState('');
+  const [searchResults, setSearchResults] = useState([]);
+
+  useEffect(() => {
+    if (searchQuery) {
+      searchCities();
```

Comment:
again I would suggest to pass search query as a param here. What if you want to move searchCities out of this component's scope

Reply:
It's true, corrected